package.json

```diff
@@ -26,6 +26,7 @@
     "bootstrap": "^4.4.0",
     "first": "0.0.3",
     "fundamental-ngx": "0.6.0",
+    "ngx-pagination": "^5.0.0",
```

Comment:
You can remove ngx-pagination library from the code

Reply:
Done